recipes/mesa/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "11.2.2" %}
+
+package:
+  name: osmesa
+  version: {{ version }}
+
+source:
+  url: https://github.com/mesa3d/mesa/archive/mesa-{{ version }}.tar.gz
+  fn: mesa-{{ version }}.tar.gz
+  sha256: a2edf7ad2589f0c3b5cd3d8f8866193c03bf871a38d5dc27423a03f352abe2de
+
+build:
+  number: 1
+  msvc_compiler: 14.0
```

Comment:
> I can try. To be clear: we're only going to build for linux then?

@ivoflipse to build this only on Linux
-  remove the `msvc_compiler: 14.0` (and any other Windows specific things)
- add `skip: True  # [not linux]` here